tools/MINGW-packages/mingw-w64-openexr/PKGBUILD

```diff
@@ -70,7 +69,6 @@ build() {
       -DBUILD_SHARED_LIBS=ON \
       -DBUILD_TESTING=OFF \
       -DOPENEXR_INSTALL_EXAMPLES=OFF \
-      -DOPENEXR_INSTALL_PKG_CONFIG=ON \
```

Comment:
Why is this and the OPENEXR_BUILD_TOOLS=OFF logic above being removed? Are then no longer needed/defined in the new version of OPENEXR?